.github/actions/publish-docs/action.yml

```diff
@@ -0,0 +1,24 @@
+name: Publish Documentation
+description: 'Publish the documentation to Github pages'
+inputs:
+  token:
+    description: 'Token to use for publishing.'
+    required: true
+  dry_run:
+    description: 'Is this a dry run. If so no docs will be published.'
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - uses: launchdarkly/gh-actions/actions/publish-pages@publish-pages-v1.0.1
+      name: 'Publish to Github pages'
+      if: ${{ inputs.dry_run == 'false' }}
+      with:
+        docs_path: launchdarkly-android-client-sdk/build/docs
```

Comment:
Should this be `launchdarkly-android-client-sdk/build/docs/javadoc`? That seems to be where the docs end up when I build them locally.

Reply:
Good catch, looks like it should be.  Will fix.

Reply:
Done